packages/manager/apps/container/src/container/nav-reshuffle/sidebar/assistance.jsx

```diff
@@ -40,16 +41,24 @@ function AssistanceSidebar() {
             url: urls.get('status'),
             isExternal: true,
           }}
-          onClick={() => {}}
-        />
-        <SidebarLink
-          node={{ translation: 'sidebar_assistance_support_level' }}
-          onClick={() => {}}
-        />
-        <SidebarLink
-          node={{ translation: 'sidebar_assistance_live_chat' }}
-          onClick={() => shell.getPlugin('ux').openChatbot()}
         />
+        {['EU', 'CA'].includes(environment.getRegion()) && (
+          <SidebarLink
+            node={{
+              translation: 'sidebar_assistance_support_level',
+              routing: {
+                application: 'dedicated',
+                hash: '#/useraccount/support/level',
+              },
+            }}
+          />
+        )}
+        {['EU', 'CA'].includes(environment.getRegion()) && (
```

Comment:
We have the two same conditions, it would be nice to extract it. 

Eventhough it is not heavy calculations, it is not likely to have different outputs during the same render for this condition. So extracting it in a const before the return, will calculate it once each render instead of calculating it on each node.